dataactcore/interfaces/function_bag.py

```diff
@@ -660,16 +661,32 @@ def get_fabs_meta(submission_id):
 def get_action_dates(submission_id):
     """ Pull the earliest/latest action dates from the DetachedAwardFinancialAssistance table
 
-    Arguments:
-        submission_id: submission ID pull action dates from
+        Args:
+            submission_id: submission ID pull action dates from
 
-    Returns:
-        the earliest action date (str) or None if not found
-        the latest action date (str) or None if not found
+        Returns:
+            the earliest action date (str) or None if not found
+            the latest action date (str) or None if not found
     """
 
     sess = GlobalDB.db().session
     return sess.query(func.min(DetachedAwardFinancialAssistance.action_date).label("min_action_date"),
                       func.max(DetachedAwardFinancialAssistance.action_date).label("max_action_date"))\
         .filter(DetachedAwardFinancialAssistance.submission_id == submission_id,
                 DetachedAwardFinancialAssistance.is_valid.is_(True)).one()
+
+
+def get_last_modified(submission_id):
+    """ Get the last modified date for a submission
+
+        Args:
+            submission_id: submission ID to get the last modified for
+
+        Returns:
+            the last modified date of the provided submission or None if the submission doesn't exist
+    """
+    submission_updated_view = SubmissionUpdatedView()
+    sess = GlobalDB.db().session
+    last_modified = sess.query(submission_updated_view.updated_at).\
+        filter(submission_updated_view.submission_id == submission_id).first()
+    return last_modified.updated_at if last_modified else None
```

Comment:
Wondering if we should have it default to `submission.updated_at` if for some reason it's not in `SubmissionUpdatedView`

Reply:
But a view is basically a virtual table. It gets updated automatically every time the elements it pulls from are updated. In order for it to be a submission and not in the view it'd have to either already not have an `updated_at` or the entirety of SQL would have to have a bug.